nixos/roles/lamp.nix

```diff
@@ -207,13 +208,14 @@ in {
         # which conflicts with our webgateway role.
         services.httpd.virtualHosts = {};
 
+        services.phpfpm.phpPackage = role.php;
+
         services.phpfpm.pools = builtins.listToAttrs (map
           (vhost: {
              name = "lamp-${toString vhost.port}";
-             value = {
+             value = lib.attrsets.recursiveUpdate vhost.pool {
```

Comment:
mkMerge would likely be better

Reply:
decided against since vhost.pool only contains overrides now